drivers/vulkan/rendering_device_driver_vulkan.cpp

```diff
@@ -4585,6 +4586,15 @@ RDD::PipelineID RenderingDeviceDriverVulkan::render_pipeline_create(
 	rasterization_state_create_info.depthBiasSlopeFactor = p_rasterization_state.depth_bias_slope_factor;
 	rasterization_state_create_info.lineWidth = p_rasterization_state.line_width;
 
+	VkPipelineRasterizationConservativeStateCreateInfoEXT conservativeStateCreateInfoExt = {};
+	if (enabled_device_extension_names.has(VK_EXT_CONSERVATIVE_RASTERIZATION_EXTENSION_NAME)) {
+		if (p_rasterization_state.enable_conservative_rasterization) {
```

Comment:
I think we should reverse the orders of these checks so we can avoid indexing into the hashset unless we need to.

Reply:
Actually should it probably warn if the extension is not available? This only runs at pipeline creation so it wouldn't spam the console.